purchase_order_product_recommendation_xlsx/wizards/purchase_order_recommendation_view.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_recommendation_view_form" model="ir.ui.view">
+        <field name="model">purchase.order.recommendation</field>
+        <field
+            name="inherit_id"
+            ref="purchase_order_product_recommendation.purchase_order_recommendation_view_form"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='price_unit']" position="attributes">
+                <attribute name="force_save">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='units_available']" position="attributes">
+                <attribute name="force_save">1</attribute>
+            </xpath>
+            <xpath
+                expr="//field[@name='units_virtual_available']"
```

Comment:
Just a little comment: In the wizard, this field shows as "Forecasted Qty":
![image](https://user-images.githubusercontent.com/38977934/116965287-57f22380-aca5-11eb-8041-ecfa8707358c.png)
But, it the exported xlsx, the column is called "Units Virtual Available":
![image](https://user-images.githubusercontent.com/38977934/116965357-7e17c380-aca5-11eb-9663-edfd9416a3eb.png)
Is there an easy way of making those fields have the same name? It may be more intuitive for the user...